npm-debug.log

```diff
@@ -0,0 +1,47 @@
+0 info it worked if it ends with ok
```

Comment:
@cch5ng can you add `npm-debug.log` to `.gitignore`, and remove this file from the PR?